.github/workflows/ci.yml

```diff
@@ -198,8 +198,6 @@ jobs:
         [Output]
         BootProtocols=uefi
         WithUnifiedKernelImages=no
-        # Boot with systemd.volatile=overlay so squashfs images can write to the filesystem
-        KernelCommandLine=systemd.volatile=overlay
```

Comment:
Here, since I can't comment on the commit message: You mean `gpt_squashfs`, right? Since we still test plain squashfs.